src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonConverter.cs

```diff
@@ -20,6 +20,15 @@ internal JsonConverter()
             ConverterStrategy = GetDefaultConverterStrategy();
         }
 
+        /// <summary>
+        /// Gets the type being converted by the current converter instance.
+        /// </summary>
+        /// <remarks>
+        /// For instances of type <see cref="JsonConverter{T}"/> returns typeof(T),
+        /// and for instances of type <see cref="JsonConverterFactory"/> returns <see langword="null" />.
+        /// </remarks>
+        public abstract Type? Type { get; }
```

Comment:
I'm curious about the naming choice to just call this "Type".  We think it's obvious enough that it's the type to convert rather than the type of the current instance or some other type?  We obviously felt a need to clarify when it was internal and called TypeToConvert; that wasn't necessary?

Reply:
I believe the terminology was picked up from the `typeToConvert` parameter on the `JsonConvert.Read` method. It makes sense there but I don't think it makes sense when scoped to the converter. The name as proposed follows the convention of the `JsonTypeInfo.Type` property.